### DISCUSSION TITLE
docs/develop/userscripts/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#362] [Page comments]
# docs/develop/userscripts/

Userscripts are JavaScript files that are executed every time the user loads a Scratch page. They can modify the document's HTML, add new buttons, customize Scratch editor behavior, and so much more.

https://scratchaddons.com/docs/develop/userscripts/

**Comment by Samq64** [comment]
Uh... why was discussion only made now?

**Reply by WorldLanguages** [reply]
Someone reacted with a thumbs up
